ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,56 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project. It deploys Kubernetes with the Antrea image and tests whether the OVS has hardware offload capabilities or not.
+
+## Workflow
+
+The CI uses three scripts to run the tests:
+
+* start_ci.sh: Build and deploy the Kubernetes test environment including Antrea.
+* test.sh: Test the hardware offload.
+* stop_ci.sh: Stop and clean the testing environment.
+
+The CI can be triggered by commenting `/test-hw-offload` on an opened pull request.
+
+## Configurations
+
+The scripts use Bash environment variables to configure its behavior. The following table shows what variables can be configured:
+
+|  Variable |  Default value |  Comments |
+|  ------ |  ------ |  ------ |
+|  RECLONE | true | Whether to clone projects again in case of single workspace |
+|  WORKSPACE | /tmp/k8s_$$ | The directory that will contain all the project components |
+|  LOGDIR | $WORKSPACE/logs | The directory where the logs should be put |
+|  ARTIFACTS | $WORKSPACE/artifacts | The directory where configuration files should be put |
+|  TIMEOUT | 300 | Timeout time in seconds for Kubernetes resources status change |
+|  POLL_INTERVAL | 10 | The interval to wait between each check for Kubernetes resources status change |
+|  KUBERNETES_VERSION | latest_stable | The Kubernetes version (or branch) to build |
+|  HARBOR_REGESTRY | harbor.mellanox.com | The docker registry to pull images from |
```

Comment:
it's spelled "registry"

Reply:
Done. Changed in the table and in the scripts.